packages/component-library-react/src/Accordion.tsx

```diff
@@ -41,6 +41,7 @@ const previousItem = <T,>(items: T[], item: T): T | undefined => {
 
 export interface AccordionSectionProps extends HTMLAttributes<HTMLDivElement> {
   headingLevel?: number;
+  className?: string;
```

Comment:
Deze interface toevoegen is niet nodig, omdat `className` al onderdeel is van `extends HTMLAttributes<HTMLDivElement>`